package.json

```diff
@@ -113,7 +113,9 @@
       "zod@<=3.22.2": ">=3.22.3",
       "nth-check": ">=2.1.1",
       "@adobe/css-tools@<4.3.2": ">=4.3.2",
-      "@babel/traverse@<7.23.2": ">=7.23.2"
+      "@babel/traverse@<7.23.2": ">=7.23.2",
+      "braces@<3.0.3": ">=3.0.3",
+      "ws": "8.17.1"
```

Comment:
```suggestion
      "ws@<8.17.1": ">=8.17.1"
```

Reply:
this way was with problems, one of the dependencies was still getting 8.17.0

Reply:
I got the same error previously. It's the lockfile struggling to resolve that resolution.

If you want, you can cherry-pick the commit https://github.com/FuelLabs/fuels-wallet/pull/1318/commits/42c3abd044b70fe62f28c4be83d4ea7f824723b5. It will fix properly.

`git cherry-pick 42c3abd044b70fe62f28c4be83d4ea7f824723b5`